grpc-example/grpc-example-scala/kubernetes/hello-impl.yml

```
@@ -0,0 +1,51 @@
+---
+apiVersion: "apps/v1"
+kind: Deployment
+metadata:
+  name: "hello-impl-v1-0-snapshot"
+  labels:
+    appName: helloworld.GreeterService
+    appNameVersion: "hello-impl-v1-0-snapshot"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      appNameVersion: "hello-impl-v1-0-snapshot"
+  template:
+    metadata:
+      labels:
+        appName: helloworld.GreeterService
+        appNameVersion: "hello-impl-v1-0-snapshot"
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: hello-impl
+          image: "hello-impl:1.0-SNAPSHOT"
+          imagePullPolicy: IfNotPresent
+          env:
+            # normally APPLICATION_SECRET would be set from the k8s secret store
+            - name: APPLICATION_SECRET
+              value: "asdf1234"
```

is the secret long enough? Play 2.7 introduced entropy checks to avoid low-quality secrets. 
```suggestion
              value: "asdf1234-just-make-it-extremely-long-because-why-not-9384756owufglsdhfgancld"
```